pkg/operator/operator.go

```diff
@@ -47,41 +42,46 @@ const (
 
 var (
 	ErrVersionOutdated = errors.New("requested version is outdated in apiserver")
+	clientTimeout      = 15 * time.Second
 )
 
+type context struct {
+	clientset   kubernetes.Interface
+	namespace   string
+	retryDelay  int
+	maxRetries  int
+	masterHost  string
+	kubeHttpCli *http.Client
+	factory     client.ConnectionFactory
+	rclients    map[string]*rookclient.RookNetworkRestClient
```

Comment:
this should probably be the rook client interface, not the network implementation of that interface.  this change would propagate through to other places i think